helpers.go

```diff
@@ -954,7 +954,7 @@ func (ctx *e2econtext) getTestDefinition(rulePath string) ([]byte, error) {
 	gbuf, gerr := ioutil.ReadFile(testFilePath)
 	if os.IsNotExist(gerr) {
 		// let's check for other files and fail if they don't exist
-		files, err := os.ReadDir(ruleTestDir)
+		files, err := os.ReadDir(path.Join(rulePath, ruleTestDir))
 		if err != nil {
```

Comment:
Maybe we need some sort of logging here that says the suite didn't find something it was looking for.